deploy/tests/network.ts

```diff
@@ -623,131 +627,322 @@ import { getNamedAccounts } from 'hardhat';
+                    expect(await vbnt.balanceOf(bntWhale.address)).to.equal(prevVBNTTokenAmount);
+                    expect(await getBalance(bnTKN, bntWhale)).to.be.gt(prevBNTKNAmount);
 
-                        expect(await bnBNT.balanceOf(bntWhale.address)).to.be.gt(prevBNBNTAmount);
-                        expect(await vbnt.balanceOf(bntWhale.address)).to.equal(prevVBNTTokenAmount);
-                        expect(await getBalance(bnTKN, bntWhale)).to.be.gt(prevBNTKNAmount);
+                    expect(await bnt.totalSupply()).to.be.almostEqual(initialTotalSupply.sub(bntAmount), {
+                        maxRelativeError: new Decimal('0.001')
+                    });
 
-                        expect(await bnt.totalSupply()).to.be.almostEqual(initialTotalSupply.sub(bntAmount), {
+                    expect(await getBalance(nativeToken, masterVault.address)).to.be.almostEqual(
+                        prevVaultTokenBalance.add(nativeTokenAmount),
+                        {
                             maxRelativeError: new Decimal('0.001')
+                        }
+                    );
+                });
+            });
+        });
+
+        describe('portal', () => {
+            describe('Uniswap V2-like pools', () => {
+                let bancorPortal: BancorPortal;
+                let linkWhale: SignerWithAddress;
+                let weth: string;
+                let link: string;
+
+                before(async () => {
+                    ({ linkWhale } = await getNamedSigners());
+                    ({ link, weth } = await getNamedAccounts());
+                });
+
+                beforeEach(async () => {
+                    bancorPortal = await DeployedContracts.BancorPortal.deployed();
+                });
+
+                enum PoolType {
+                    UniswapV2 = 'UniswapV2',
+                    SushiSwapV1 = 'SushiSwapV1'
+                }
+
+                const testUniswapV2Migration = (type: PoolType) => {
+                    describe(type, () => {
+                        let router: IUniswapV2Router02;
+                        let factory: IUniswapV2Factory;
+                        let poolToken: IERC20;
+                        let poolTokenAmount: BigNumber;
+
+                        let depositedNativeTokenAmount: BigNumber;
+                        let depositedLinkAmount: BigNumber;
+
+                        const NATIVE_TOKEN_DESIRED_AMOUNT = toWei(10);
+
+                        beforeEach(async () => {
+                            const { uniswapV2Router02, uniswapV2Factory, sushiSwapRouter, sushiSwapFactory } =
+                                await getNamedAccounts();
+
+                            await ethWhale.sendTransaction({
+                                value: NATIVE_TOKEN_DESIRED_AMOUNT,
+                                to: linkWhale.address
+                            });
+
+                            await stabilizePoolV3(NATIVE_TOKEN_ADDRESS, ethWhale);
```

Comment:
not necessary to stabilize before depositing in v3

Reply:
👍